src/plugins/ISOTimeFormat/pluginSpec.js

```diff
@@ -0,0 +1,73 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2020, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+import ISOTimeFormat from './ISOTimeFormat.js';
+import {
+    createOpenMct,
+    resetApplicationState
+} from 'utils/testing';
+
+describe("the plugin", () => {
+    const ISO_KEY = 'iso';
+    const JUNK = "junk";
+    const MOON_LANDING_TIMESTAMP = -14256000000;
+    const MOON_LANDING_DATESTRING = '1969-07-20T00:00:00.000Z';
+    let openmct;
+    let isoFormatter;
+
+    beforeAll(() => {
+        resetApplicationState();
+    });
+
+    beforeEach((done) => {
+        openmct = createOpenMct();
```

Comment:
`openmct` is not used in these tests, so no need to instantiate it.

Reply:
Done and done!